### PR TITLE
Replaces Double Agents with Internal Affairs

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
del: Removed double agents mode.
add: Replaced it with Internal Affairs mode.
/:cl:

Double Agents have been rebranded as Nanotrasen Internal Affairs. Will be easiest just to paste their intro text to explain.

You are the Nanotrasen Internal Affairs Agent!
Your target is suspected of [crime], and you have been tasked with eliminating them by any means necessary to avoid a costly and embarrassing public trial. While you have a license to kill, unneeded property damage or loss of employee life will lead to your contract being terminated.
For the sake of plausible deniability, you have been equipped with an array of captured Syndicate weaponry. PDA blah blah blah uplink code
Finally, watch your back. [target] has friends in high places, and intel suggests someone may have taken out a contract of their own to protect them. 

tl;dr You are a double agent but meant to focus on your target and not kill wantonly. All agents know its double agents. This PR is obviously dead in the water unless headmins want to make them an exception to rule 4.

Why: DA is massively unpopular because it just turns into everyone killing everyone. It doesn't really function as anything other than traitor with more traitors right now. Since there isn't much worth salvaging from the mode currently I figured it'd be a good place to test if solo antags with restrictions are at all fun in our ruleset without risking a mode people actually care about.

### CODE REVIEW

**Comment by kevinz000** [comment]
Please yes.

**Comment by Incoming5643** [comment]
I see absolutely no reason why one has to replace the other.

File: code/game/gamemodes/traitor/double_agents.dm [hunk 1]
```diff
@@ -100,3 +102,18 @@
 
 		// If any check fails, remove them from our list
 		late_joining_list -= M
+
+
+/datum/game_mode/traitor/internal_affairs/greet_traitor(datum/mind/traitor)
+	var/crime = pick("distribution of contraband" , "unauthorized erotic action on duty", "embezzlement", "piloting under the influence", "dereliction of duty", "syndicate collaboration", "mutiny", "multiple homicides", "corporate espionage", "recieving bribes", "malpractice", "worship of prohbited life forms", "possession of profane texts", "murder", "arson", "insulting their manager", "grand theft", "conspiracy", "attempting to unionize", "vandalism", "gross incompetence")
```
Comment (QualityVan) [inline code/game/gamemodes/traitor/double_agents.dm]: prohbited should be prohibited

**Comment by xxalpha** [comment]
Great """"""""design"""""""" decision.

**Comment by Ausops** [comment]
Well if you want to make it an exception to rule 4 it has to state in big letters that you can't. Maybe have it explicitly spell out what we're hoping players will do. There's a chance some players will understand and appreciate it if given the reasoning behind it.

File: code/game/gamemodes/traitor/double_agents.dm [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
 
 	traitors_possible = 10 //hard limit on traitors if scaling is turned off
 	num_modifier = 4 // Four additional traitors
 
-	announce_text = "There are double agents trying to kill each other!\n\
-	<span class='danger'>Traitors</span>: Eliminate your targets and protect yourself!\n\
-	<span class='notice'>Crew</span>: Stop the double agents before they can cause too much mayhem."
+	announce_text = "There are Nanotrasen Internal Affairs Agents trying to kill each other!\n\
+	<span class='danger'>IAA</span>: Eliminate your targets and protect yourself!\n\
+	<span class='notice'>Crew</span>: Stop the IAA agents before they can cause too much mayhem."
```
Comment (Incoming5643) [inline code/game/gamemodes/traitor/double_agents.dm]: Not your fault and will literally never matter, but this encourages rushing the shuttle.

Comment (coiax) [inline code/game/gamemodes/traitor/double_agents.dm]: Surely by that logic all modes encourage rushing the shuttle.

**Comment by Tacolizard** [comment]
> Finally, watch your back. [target] has friends in high places, and intel suggests someone may have taken out a contract of their own to protect them.

What exactly does this mean? Do some Internal Affairs agents get tasked with protecting people?

**Comment by Tacolizard** [comment]
> While you have a license to kill, unneeded property damage or loss of employee life will lead to your contract being terminated.

1. this needs to be defined pretty clearly by admins.
2. Giving IAA full access to syndicate weapons could be a very bad idea with this policy. What is to stop an IAA from buying a ton of bombs and bombing their target? IMO, IAA should only have access to stealth-focused weaponry. There's no reason an IAA should be able to buy a bomb. They should only have access to things like poison, edaggers, airlock charges, and silenced stetchkins.

**Comment by GunHog** [comment]
This sounds like an admin's nightmare, given the exception to rule 4. Does this mean that IA agents are not true 'antags', and will be banned if they kill people as they please? Things could get pretty hairy if the IA agent kills people in self-defense, even for things like the clown slipping him (because even that is enough to score a kill when combined with a decent melee weapon). Also, at what point are the agents "valid"? Is Sec supposed to support them or kill them?

**Comment by TheColdTurtle** [comment]
Will this still allow Ai's to become double agents or internal affair agents? Will the flavor text be any different?

**Comment by coiax** [comment]
@GunHog I hate the notion of a "true antag", and feel there should be more restrictions on what antags can do.

**Comment by XDTM** [comment]
Sec is supposed to stop them, according to forum discussion.

**Comment by Incoming5643** [comment]
a lot of these problems are server problems, not codebase problems

**Comment by GunHog** [comment]
>@GunHog I hate the notion of a "true antag", and feel there should be more restrictions on what antags can do.

I dislike the idea of this entirely. I am a strong supporter of Rule 4, and do not like the idea of _additional_ restrictions to primary antags. Part of why I stay on /tg/ is because of lax rules on antags.

**Comment by KorPhaeron** [comment]
@ausops

I put the "dont kill randomly or your contract gets terminated" in giant red text but I could make it more explicitly OOC

@gunhog

They are like abductor victims in that they can kill in pursuit of their goals but not at random.

Obviously this will create grey areas but I think that shooting a med doctor for his ID so you can sneak into medbay and get your target or killing a witness is vastly different than max capping the station or running down the hall with a double esword.

They are disguised as syndicate so sec will still stop them.

**Comment by Bawhoppen** [comment]
I know double agents isn't good, but can we get a proper rework instead of half-assed change? I had a design doc written up on it if you're curious

**Comment by XDTM** [comment]
Preventing agents from just going apeshit on the crew may not be a huge code change but it's pretty big in gameplay

**Comment by lzimann** [comment]
Just scrap this mode entirely.

**Comment by lzimann** [comment]
Also huge :-1: on "now you have limits on antag in this gamemode"

**Comment by XDTM** [comment]
Team antags have limits already, it's not such a new thing. And not just "don't kill your team", but stuff like no releasing the engine or getting the shuttle called during cult.

**Comment by Bawhoppen** [comment]
:-1:x:100: from me if it wasn't clear

**Comment by bgobandit** [comment]
@KorPhaeron is it possible to pull crime from the stock market crime list or, better yet, move that to a separate strings list, just to limit the duplication of stuff

**Comment by Robustin** [comment]
Q1: If DA is "hugely unpopular", why is it weighted so heavily?

Q2: Now that agents are "loyal", how exactly do ordinary crew participate in the round? A group of 12 dudes who can only murder each other, if youre not in the circle of KILL then it might as well be extended for you. 

If we were higher-RP then Q2 wouldn't be an issue because you could plausibly have traitors posing as Internal Affairs and whatnot. When random killing is explicitly prohibited, it will be extremely easy to metagame whether its actually an IA round and people will quickly learn that bluffing is pointless and correspondingly, people claiming to be IA agents are almost certainly telling the truth. 

So what happens when the crew invariably realizes that its an IA round and they are effectively playing extended?

**Comment by XDTM** [comment]
This is only a conjecture, but i think that IAA are going to be able to cause some collateral damage/eliminating key witnesses as long as they're doing it in function of their objective and not for shits and giggles.

Also DA is probably weighted heavily because we keep a balance between solo and team antag, and since wiz and blob have to be rare it's gotta be traitor, traitorling or DA. Plus even if a gamemode is hated it takes a long time to actually get reduced or removed from the rotation, gang was despised for several months before going out.

**Comment by Robustin** [comment]
If we don't have strict rules about collateral murder then IAA just becomes an ahelp shitfest and admins have to become bay-style RP police with a huge grey area of what level of collateral is permitted. 

Its just not justiciable to have admins decide on some arbitrary level of acceptable collateral. It sounds like Kor wants it to be strict too, but that raises the problems I discussed above.

**Comment by XDTM** [comment]
A grey area is not necessarily bad, because a clear line is very subject to line-toeing and abuse. Plus, reading the forums you can see that adming rulings may vary already, since we lack clear-cut rules in various areas of the game, like escalation.

**Comment by ChangelingRain** [comment]
This is very good but it should be clear to both the agents AND the admins that they can't murder everyone and should perhaps refrain from drawing so much attention that they need to do so.

**Comment by Repukan** [comment]
Why not just put the nail in the coffin, add anti murderbone rules and name both servers nu-yogstation.

👎

**Comment by KorPhaeron** [comment]
@XDTM 

Comparing this to escalation rules is the single strongest argument I have heard against this

**Comment by Robustin** [comment]
There isn't any wiggle room for "you can't deliberately kill anyone who isn't your target unless it would be acceptable under ordinary escalation rules" that doesn't already exist. 

Adding grey to that just means a lot of headaches for players and admins alike. Even @ChangelingRain  illustrates how short-sighted people are viewing this. She says its "clear they can't murder everyone", ok sure we know that isn't allowed, but what about murdering everyone in your department so that there won't be any witnesses when you try to lure your target there to kill them? Can I always bomb genetics under the rationale that I don't want to risk my target getting cloned? Can I ebow + esword everyone I see in maint because they might be targeting me and/or they might find where I stashed the body?

**Comment by Repukan** [comment]
Can I kill all of sec because they might try to ruin my peaceful life

**Comment by XDTM** [comment]
@KorPhaeron good point, but with clearly defined rules you'd have arguments like @Robustin just proposed, which are thinly veiled excuses to murderbone that still fall into the 'You can eliminate witnesses' rule (which i'm not sure if exists or not).

**Comment by KorPhaeron** [comment]
Yeah but escalation rules have suffered from a steady creep into de facto removal when certain admins are online

**Comment by Robustin** [comment]
I'm not sure you read the first paragraph there @XDTM 

We COULD have a very clear line drawn, but that would also turn this mode into extended for anyone who's not in the murder circle-jerk. Why would sec even care about IAA's killing each other? At that point IAA's are basically a security "my job is to neutralize antags" officer.

**Comment by coiax** [comment]
The notion that a bunch of people playing spy vs spy and killing each other would have no affect on the round is bemusing, but wrong. There is always collateral damage.

**Comment by Metallusions** [comment]
Some suggestions: 
1) during DA rounds, until at least one half of all IAA pairs on the station are dead or off-station, Nanotrasen subtly recalls the shuttle before the cutoff (tracing to an on-station comms console by Unknown) while stating the reason why at a communications console- a litany of excuses is used to say that whatever departments still have agents with unfinished business have fallen behind on their quotas or whatever, prompting command and sec to help resolve the standoff one way or the other in order to leave.
2) if IA agents are working for Nanotrasen, then Security is a potential department for double agents as they could conceivably complete their task while remaining loyal to the company.  WITH the caveat that they don't get an uplink/have reduced TC.
3) edit: people already brought up a 'no witnesses' flavortext

**Comment by KorPhaeron** [comment]
I think he's saying he'd purposefully cause collateral to the point that the rule would be meaningless, until admins cracked down so hard that even actual collateral would be swept up in enforcement and we are left with parapens only final destination

**Comment by Robustin** [comment]
As soon as the metagame around the mode had formed, it wouldn't be a "notion". People would routinely announce that they are agents, the lack of murderboning would also make it really obvious that it was IAA. Once an IAA round is effectively confirmed, people have 0 reason to give a shit about murders "OH NO THEY MURDERED SOMEONE WHO WAS GOING TO MURDER SOMEONE WHAT A TRAGEDY". 

Kor I just had a lightbulb moment:

DA is typically 12 traitors now. Make IAA into 6 traitors + 6 IAA. You could enforce the no collateral rule against agents strictly but the presence of traitors would prevent the cancerous metagame from forming and inevitably suffocating the mode into extended.

**Comment by coiax** [comment]
> cancerous metagame

What is this bullshit. People announce they're traitors all the damned time, you literally scream codewords on the radio so much, people are starting to do it to meme. The notion of dunk the antag is so ingrained, people will kill anyone holding an esword.

**Comment by Robustin** [comment]
Are you purposely reading that as dimly as possible Coiax? The whole point is that traitor is the "Default" metagame and so it doesn't matter if people think traitors are on board. The traitors are necessary to have an IAA round maintain the "Default" mindset of actually caring about murder, people with ebows, etc.

If IAA is traitor-free and murderbone-free, then the metagame is no longer "default" and people learn to treat each other differently just like they would during a wizard/warops/blob round.

**Comment by coiax** [comment]
@GunHog if you don't want this to happen, you should ask Kor if you can remake the PR as the author. :D

**Comment by GunHog** [comment]
Is there not enough salt in your tanks already, coiax!?!?!?!?!?

**Comment by coiax** [comment]
<3

**Comment by Bawhoppen** [comment]
robustin did you just say you had a lightbulb when that's literally like a generic idea and I brought it up in the coderbus WHILE YOU WERE THERE like an hour or two ago

**Comment by Militaires** [comment]
i like this cause it encourages stealth which is what i really used to love about tg combat

those clutch moments with hiding, pulling off that murder silently and then fucking up but then you quickly taking out/framing the witness who saw it.

that kind of gameplay has sadly **disappeared**, and the way this encourages it is splendid.

:+1: 

give IAA cqc

**Comment by Robustin** [comment]
>Believing a mode where nobody has any reason to mess with antags will encourage stealth

The only stealth that will happen is hiding in lockers once your target is dead because the antag is no longer an antag once their target is gone.

**Comment by Militaires** [comment]
@Robustin 

no the stealth will happen because you cant roundstart bomb 10 people and run around with an esword murderboning innocents, youll have to downplay it because you cant let everyone know you're an antag,  because you cant kill everyone.

**Comment by Robustin** [comment]
Militaires have you never encountered the "friendly antag"?

Back when DA's KNEW they were DA's people would routinely buddy up with security. This is the same issue only 100x worse because now the IAA cant backstab security and murder them all and security has an objective measure for weighing if someone is telling the truth (i.e. its highpop with syndie gear and nobody is murderboning yet). If the crew can "tell" that its an IAA round then they have no reason to give a fuck about one agent killing another. 

Thats why it needs a healthy mix of actual traitors.

**Comment by SgtxRootbeer** [comment]
Actually, just remove their syndie uplink so they have to rely on stealth to survive.

**Comment by ExcessiveUseOfCobblestone** [comment]
robustin is right, you guys should take him off your ignore list :^)

**Comment by Hatfish** [comment]
Perhaps don't restrict who the IAA can kill but instead add an objective to not kill above a certain limit?

**Comment by Militaires** [comment]
@hatfish 's idea is actually damn good.

**Comment by InsaneHyena** [comment]
How do you even measure how many people IAA has killed?

**Comment by Militaires** [comment]
@insanehyena 's point is actually damn good.

**Comment by Incoming5643** [comment]
There's not an easy way to properly document how many people someone has killed. There is a huge grey area for what counts as a "kill", and nothing attempts to clear that up. You would probably have to refactor attack logs to track that and even then thanks to how abstract things can get with being in crit it still probably wouldn't be that accurate.

**Comment by BeeSting12** [comment]
I don't like putting restrictions on antags at all. 👎  from me.

**Comment by BeeSting12** [comment]
Like the concept makes more sense than the other thing but it's still dumb if it puts restrictions on antagonists. Part of ss13 is that wiggle room that allows for fun stories.

**Comment by KorPhaeron** [comment]
But we keep telling the same story which is "spree killer kills everyone in workplace" to the point where other people are having trouble getting stories in

**Comment by Militaires** [comment]
lol fun stories

if you consider esword 2x every round then **Very Well.**

**Comment by Repukan** [comment]
This is just gonna end up like yog where if you kill one of the admins or admin buddies for whichever reason they dogpile on you and stick notes because God forbid you kill beyond your target.

Granted our admins are not kids like the ones in yog but it could in a way devolve into that.

**Comment by Militaires** [comment]
people say that antags in tg have no restrictions but that's not true because..

when am i going to be able to skullfuck an assistant then tie them up for public use??????????????????????????????????????

WHEN????????????

**Comment by KorPhaeron** [comment]
@Repukan

Legitimate concern given how IC issues slowly got ground away and notes snowball so hard already

**Comment by Repukan** [comment]
>He doesn't play during the time of night when both servers are as free of laws as international waters

**Comment by Repukan** [comment]
If you want to see what limiting antags looks like just observe rounds in yog.

It's actually fucking disgusting because Adam has a direct line to the admins. If deadchat whines enough and or he dies they just call an admin on, make him check the antag objectives and BSA them + note. I saw this once and I thought "wee I'd rather stay on tg."

If people keep getting caught just double eswording then reduce their chances of being antag or give them a talk. This shit could well leak into cults later on because it creates a toxic mentality of "wee lad I better not do anything and hope my target dies alone rather than acknowledge this admin breathing down my neck."

**Comment by Repukan** [comment]
Even more so a mode like this is contrarian as fuck given that we have at least three gamemodes were murderboning is necessary for victory in some cases (Wizard, war ops, rev or blood cult.)

I'd feel there's a better way to solve the anti-win mentality issue but this change is definitely not a healthy way to do it.

**Comment by Shapsy** [comment]
The fact that certain people here can't differentiate between "killing a few people out of perhaps questionable necessity" and "loudly announcing to everyone that you're going to murder the entire station with a double esword and the entire armory" is silly. If you don't think admins can tell the difference between killing a few people (perhaps with some leeway that would allow for _some_ FNR murders slipping through) and killing the entire station time and time again, you're being ridiculous.

>Even more so a mode like this is contrarian as fuck given that we have at least three gamemodes were murderboning is necessary for victory in some cases (Wizard, war ops, rev or blood cult.)

this is not an argument

**Comment by PKPenguin321** [comment]
@KorPhaeron like ausops said, the rule 4 exception would have to be notable. i think you could maybe do it by adding an actual objective that just defaults to Success that says "Avoid unnecessary station damage or causalities." this actually should also be added to abductors if we're gonna enforce that too

**Comment by PKPenguin321** [comment]
I do like this idea by the way, I've yet to read everything here but I think this preserves the original intended feel of double agents working against each other while also effectively handling the current problem of "it's just traitor but with 3x the antagonists".

**Comment by ExcessiveUseOfCobblestone** [comment]
> The fact that certain people here can't differentiate between "killing a few people out of perhaps questionable necessity" and "loudly announcing to everyone that you're going to murder the entire station with a double esword and the entire armory" is silly.

I think they're very aware of this, it's just that they'd rather not have to deal with Admin roulette when the waters get a bit muddy.

**Comment by Robustin** [comment]
Shaps is also way too overconfident in their judgment and the ability of the adminbus to be consistent on such an incredibly vague standard. **Its pure fallacy to say that just because we can all tell the difference between some TurboHitler racking up a 20 man killstreak and the guy who does his objective and then goes into deep cover that somehow the 99.9% of cases that fall in between will be similarly easy to differentiate.**

**Comment by Shapsy** [comment]
Admins wouldn't be holding people accountable for each and every kill, but if someone starts getting too kill happy and slaughters 4-5 people for funsies, admins would be able to step in and handle the situation from there. Admins wouldn't be straight banning people for every extra kill, they'd just have grounds to step in and ask someone to reign it in when they start getting needlessly bloody.

And of course, context would be very important. A normally very nice player or someone with no notes bumbling into a situation and accidentally having to off a few people to get out of a tight situation wouldn't raise much alarm, but someone who openly declares that they're going to fight the spirit of this one mode and toe the line as much as possible would be much more likely to be given a second look when they keep """accidentally""" winding up in positions where they have to kill lots of people.

You're all thinking this will lead to straight bans where players and admins are directly at each others throats, there's simply no reason to think that a quick adminpm couldn't sort out an issue without further harm.

**Comment by BeeSting12** [comment]
If we're going to replace double agents with some shitty version with antagonist restrictions then I'd rather just have the mode gone. In fact lets get rid of it entirely because it would be bad either way.

**Comment by Robustin** [comment]
Right because "dont let people get too killhappy" is going to be applied fairly and consistently.

5/6 of my last ahelps went to IRC, when someone does respond they have 0 context outside of the ahelp, I don't envy anyone who gets caught in a mess of "DID AN IAA KILL TOO MANY PEOPLE? READ THESE 6 PAGES OF LOGS AND BWOINK 10 DIFFERENT PEOPLE TO FIND OUT!"

Plus I dont even think, even if we did try to implement a shitty rule, that it would be about the quantity. Maybe the IAA kills someone in a really public manner and then gets falsely accused of flooding plasma by their hunter. Soon he's got half the station chasing him around? Can he just start cutting people to pieces until they stop fighting back?

Killing 20 people in that situation sounds far more justifiable than "I'm an IAA and I saw Phoebe Lutsu outside medbay and I hate that skank so I put 4 revolver rounds in her dome even though my target was already dead and incinerated."

**Comment by Bawhoppen** [comment]
robustin is right

**Comment by Bawhoppen** [comment]
the best solution right now is to remove the mode entirely

**Comment by Ausops** [comment]
Headmin majority is in favour of at least trialing IA. I believe it will work, at least with the aim being gathering experience and data on how removing rule 4 works out. The trials will also happen under the following conditions,

> IA is only actually run when an admin is available to supervise it.
> Admins will rarely ban or note and instead give gentle advice, encouragement or warn players. I know things will be inconsistent and a trial like this needs positive rather than negative enforcement. Admins will also be encouraged to handle things IC. If someone chooses to murderbone the station, admins have a variety of options, not limited to changing the objectives of other IA agents to assassinate the murderboner to more traditional options like sending an ERT team to get rid of the rogue IA agent.

A lot of this depends on the individual admin and how willing they are to invest and commit time into stewarding an IA round so we'll see.

**Comment by Ausops** [comment]
To robustin and similar-minded folk, try to view IA not as the next change happening to /tg/ but a series of admin events with a more ''''''''rp''''''''''''' focus in mind. Think less of enforcement and more admins encouraging players to try out a different playstyle in a few rounds each day or possibly just a few rounds each week.

**Comment by Repukan** [comment]
We do have an rp oriented mode, it's called devil and it's sitting in the bottom of the thrash can, a forgotten testament that code cannot solve mentality.

**Comment by Magikarpet** [comment]
Devil has a major issue that doesn't correlate to roleplay and that's what it provides with people who sign contracts

you know better than I that having Dindu Mcmuffins run around with fireball and knock as a nonantag isn't going to end well due to either sec or other antags butting heads with them

that also ignores the fact that a fully staged and evolved devil shits out slaughter demons that can't bloodcrawl only limited by the number of ghosts on at that time

**Comment by Repukan** [comment]
Well then why was it promoted with "RP" as the main focus?

It was a shit mode that got merged because there were promises of "good roleplayers will be given this role!" And "Oh boy I can't wait for my wacky situations!" Which ended up not happening and just were a faux pass for faux-conversion antag to get merged too.

It was also promised "oh yeah people will have to follow their sin!" But we both know that Dindu McMuffins only wanted powers

**Comment by Incoming5643** [comment]
It was made because I played "The Ship" and thought the concept could be cool in space.

And it can be cool in space, it's just that people figured out that the most effective way to kill someone is usually to kill everyone.

**Comment by KorPhaeron** [comment]
The Ship penalizes you if you kill randomly which I found out very quickly by trying to play it like SS13

**Comment by Incoming5643** [comment]
It could be simulated here by giving DAs some protect objectives on the side, but the notion of people just randomly keeling over would still be an issue here. Also it would once again reveal the difference between DA and traitor rounds.

**Comment by CosmicScientist** [comment]
I don't think this could work effectively in a game that doesn't or cannot reliably track kills and reacts automatically, especially in a role playing environment where a DM hovering over your every action may be better suited (but we can't have an admin for each agent and it's not always fun to watch a spaceman faff around to only die to someone carrying a circular saw forty minutes in when they finally do something).

When I first saw this, I was confused as to if this meant sec wasn't supposed to ~execute~ on the spot bludgeon them to death and space the corpse. Whilst I can see that this is intended as an advanced experiment, I don't know if the current branding would be good in future if this is to stick around. Either new time players will be confused or it'll add to the *!FUN!* of spessmen and just be a learning experience.

You might as well have double agent pop up with no bombs + tell the agents it's DA *whilst* beforehand asking online admins if they want it to be DA/extra scrutiny mode and if there are no admins or none respond in time, just swapping to a different game mode.

The only other thing is that I'm not sure if this will be any good. Unless I've always gotten rotten luck, DA to me is just disguise forever to not die + never find your target because they disguised. I've had a couple of fun rounds where I didn't disguise but I'm not sure if knowing I may just get double tapped by a revolver if I leave/whilst I forever stay in a populated area and then debrained is any fun at its worst. I guess if I'm told beforehand I might decide to be a dick and see if I can catch out who's trying to kill me (and hope I don't get bwoinked for pulling out a weapon on someone who appeared to follow me around and didn't speak a word, *whilst* running away from sec/the crew because now I'm a known murderer/valid).

I do like the fluff reasons for why you're murdering the spessman.

**Comment by uraniummeltdown** [comment]
This is a mistake, if we wanted to play traitor but be forced to only do objective-related stuff there's always Yogs and Paradise

**Comment by ExcessiveUseOfCobblestone** [comment]
> The Ship penalizes you if you kill randomly which I found out very quickly by trying to play it like SS13

The Ship also doesn't let you defend on someone else's behalf, or even your own really unless the guy swinging at you has you as their target.

**Comment by KorPhaeron** [comment]
Not saying SS13 is the exact same as the ship, just that you will never get spy vs spy without guidelines.

@uraniummeltdown

We did the same thing with abductors rulewise and we didnt turn into yog

**Comment by Anonmare** [comment]
I still think you want some actual Syndicate agents in there so security can't take anyone at their word when they "It's okay I'm Internal Affairs"

**Comment by Robustin** [comment]
I'm fine with testing it, I just hope people realize that after the initial "huh whats going on?" period, this mode is going to fall prey to serious metagame issues. Whether that's a shitty unjusticiable standard of antag behavior or a stale extended mode for non-antags is up to the administration. 

I'll keep saying it until I'm blue in the face: Make it roughly 50/50 traitors/IAA and most of these problems go away. You get a traitor mode but with less murderboning and more intrigue and that's all we seem to want from this anyway.

**Comment by uraniummeltdown** [comment]
@KorPhaeron Abductors are just a side antag though, only appearing in the middle of rounds
With restricted roundstart antags it'll turn into a 💤fest

**Comment by Crocodillo** [comment]
@Robustin has a great idea in mixing IAA/traitors. If some traitor starts murderboning the IAAs would be encouraged to deal with them too, since they're supposed to be NT loyal. You'd basically have an undercover NT agent working for the good of the station. But if it's IAA only it's going to get meta gamed every single round and as soon as someone figures out it's IAA and screams it over the radio it's over. By having a mix IAA and traitors will be indistinguishable so you'll never know who is who.

**Comment by Qbopper** [comment]
holy shit IA has a PR

I don't care what changes or what needs to be done MERGE THIS

**Comment by sawrge** [comment]
Yo add me on snapchat @ytkilla

**Comment by Militaires** [comment]
just make it so it's **ENCOURAGED** not a horrible **_RULE 4 VIOLATION_**.

Just make it encouraged enough that the players think it's a rule!

**Comment by KorPhaeron** [comment]
If it ends up super boring we can add traitors but spy vs spy will fall apart if the traitor just does a 5 minute supermatter meltdown or maxcaps the station

**Comment by Robustin** [comment]
All that matters is the law of large numbers and averages when it comes to tossing traitors into a mix. 

Rounds fall apart and lose coherence, it happens, sometimes thats a good thing. The whole point is that people are pretty happy with the chaos of ordinary traitor rounds and having a round with even LESS traitors shouldn't be a problem for having too much chaos if rounds can generally stay coherent during traitor as it is. 

I don't see how pure antag v. antag will not be super boring if we know they can't, you know, antagonize the rest of us.

**Comment by Incoming5643** [comment]
This was far from settled. Was it even test merged once?

**Comment by Robustin** [comment]
No of course not. Maintainers need not worry themselves with the concerns of us mortals.

**Comment by ChangelingRain** [comment]
please don't associate innocent maintainers with this reckless fruit

**Comment by Cheridan** [comment]
How is 30 upvotes 6 downvotes "far from settled"

**Comment by CosmicScientist** [comment]
>please don't associate innocent maintainers with this reckless fruit
Bahaha! Nice joke.

>How is 30 upvotes 6 downvotes "far from settled"
Search by merged + most downvotes. There are arguments for merging those despite frivolous emojis. There are arguments for not merging this despite upvotes.

For example I still think this is useless without demanding an admin being online and not AFKing until a bwoink happens because the entire point is moderating behaviour that the computer can't catch. That and this is a "test" so, well, was this test merged? I know this isn't a serious repo and there's nothing much to worry about but be real.

**Comment by KorPhaeron** [comment]
>arbitrary number of reddit emojis approximating player reaction supports it
>headmins support it
>admins support it
>coders support it
>headmins said it wont be in the rotation and instead run sparingly as an event in recognition it might not work well
>US MORTALS IGNORED HUH

Epic

**Comment by Robustin** [comment]
@KorPhaeron 

I initially assumed that oranges went around the Headmin vote because they (Ausops) said in a very clear statement that "Headmin majority is in favour of at least trialing IA. "

Logically that carries a very clear meaning. Also since Headmins only have authority to Test-Merge I assumed that meant they would be test-merging this as part of the trial. 

So I think many of us reasonably expected to see this tested before merging.

Now we've permanently removed DA and added a new mode as part of a "trial" and the only way the trial can be "terminated" is by opening a revert PR - where the headmins have no special authority. So why did Ausops even call it a trial when he basically meant the opposite.  Unless there's been some secret agreement to auto-revert if Headmins request it, this just feels like a plain-old full merge with no strings attached.

**Comment by CosmicScientist** [comment]
tfw too mortal to notice Ausop's comment. I bow out.

**Comment by KorPhaeron** [comment]
They can also terminate the trial by refusing to enforce any rules around it, at which point this becomes traitor, or just removing it from the rotation and replacing it with traitor. Headmins ultimatley have veto power over any mode (see: gang).

Even assuming a completley adversarial relationship between coderbus and adminbus in which I decided to take the flavour text of DA hostage there isnt actually absolute seperation.

It's also more convienent to have things merged if you want to test them long term.

**Comment by Robustin** [comment]
Yea thats why I felt trial was misleading. If IAA is now in "trial" then so is Gang War, Cult, Clock Cult, Blob, Wizard, and anything else that isnt Traitor or Nukeops. Headmins have always had authority to effectively remove any mode from the game and have historically done so when the mode didn't pan out. 

That aside, the removal of DA was a critical part of this PR and it's definitely not being "trialed". DA is out and headmins have absolutely no power to bring it back if the trial fails.

**Comment by duncathan** [comment]
shut the fuck up about useless semantics god damn
surely there's a feedback forum you can yell into somewhere

**Comment by Bawhoppen** [comment]
I just don't think it's the right execution

**Pull request merged**